config/database.yml

```diff
@@ -5,6 +5,8 @@ development:
   adapter: postgresql
   database: theodinproject
   host: localhost
+  username: 
```

Comment:
Thanks, yeah. I don't recall ever staging it for a commit, so I was surprised to see this myself? Will get on it ASAP.